File: algorithms/ar-min/c/ar_min.c
#include<stdio.h>

int main(){
    int arr[7] = {2,4,7,1,3,6,5};
    int len = sizeof arr / sizeof arr[0];
    int min = arr[0];
    for(int i=1;i<len;i++){
        if(arr[i]<min){
            min = arr[i];
        }
    }
    printf("%d\n",min);
    return 0;
}

File: algorithms/ar-min/python2/ar_min.py
import sys

arr = []
minarr = sys.maxint
print "Enter the number of elements : "
n = int(input())
for i in xrange(0, n):
    num = int(input())
    arr.append(num)
    if num < minarr:
        minarr = num

print("Min element in array ", arr, " is ", minarr)
